releasenotes/notes/aiohttp_jinja2-25d9a7b4e621fad2.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    aiohttp_jinja2: move into new integration. Formerly the aiohttp_jinja2 instrumentation was enabled using the aiohttp
```

Comment:
This is technically a breaking change but I don't think we have a good migration story here unless we wanna just call out that it's deprecated in 0.x and squeeze it into the 1.0 release

@brettlangdon @mabdinur @majorgreys

Reply:
what will break is anyone doing `patch/_all(aiohttp=True/False)`

`aiohttp=False` is probably the worse breaking change for people.

or if someone is manually importing `ddtrace.contrib.aiohttp.patch` directly.

Reply:
Decided offline that we'll maintain this behaviour for now until we address it in the version policy 
#3332